### PR TITLE
LF-107: Add new nxtp executor that uses getStepTransaction endpoint

### PR DESCRIPTION
**Pull request by mathiasmoeller**
[LF-107](https://lifi.atlassian.net/jira/software/projects/LF/boards/1?selectedIssue=LF-107)

# TODO:
- [x] Implement continue logic in `nxtp_new.execute.ts`
- [x] add cbridge icon

Later
- [x] Check which parts of the old implementation are used by Xpollinate and remove the rest

### CODE REVIEW

File: src/components/Swapping.tsx [hunk 1]
```diff
@@ -397,6 +419,9 @@ const Swapping = ({ route, updateRoute, onSwapDone }: SwappingProps) => {
           case 'horizon':
             avatar = horizonAvatar
             break
+          case 'cbridge1':
+            avatar = 'cBridge1'
```
Comment (mathiasmoeller) [inline src/components/Swapping.tsx:423]: Do we have a proper avatar for this?

File: src/components/Swapping.tsx [hunk 2]
```diff
@@ -526,7 +566,7 @@ const Swapping = ({ route, updateRoute, onSwapDone }: SwappingProps) => {
       // lifi supported?
       if (isLifiSupported(route)) {
         if (!steps[0].execution) {
-          triggerLifi()
+          // triggerLifi()
```
Comment (mathiasmoeller) [inline src/components/Swapping.tsx:569]: Is this on purpose?

File: src/services/hop_new.execute.ts [hunk 3]
```diff
@@ -0,0 +1,126 @@
+import { TransactionResponse } from '@ethersproject/abstract-provider'
```
Comment (mathiasmoeller) [inline src/services/hop_new.execute.ts:1]: Can't we replace the old hop implementation? (`hop.execute.ts`)

File: src/services/cbridge1.execute.ts [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    const personalizedStep = await personalizeStep(signer, step)
+    const { tx: transactionRequest } = await Lifi.getStepTransaction(personalizedStep)
+
+    // STEP 3: Send Transaction ///////////////////////////////////////////////
+    crossProcess.status = 'ACTION_REQUIRED'
+    crossProcess.message = 'Sign Transaction'
+    update(status)
+    if (!this.shouldContinue) return status
+
+    const tx = await signer.sendTransaction(transactionRequest)
```
Comment (mathiasmoeller) [inline src/services/cbridge1.execute.ts:52]: Here the check if the transaction was already sent (`crossProcess.txHash` exists) is missing, right?

**Review by Addminus — APPROVED** [review]

**Pull request merged**